AnnoyancesFilter/sections/push-notifications.txt

```diff
@@ -524,6 +524,8 @@ ifunny.co##.push-notification
 ||news.biglobe.ne.jp/js/push/
 news.biglobe.ne.jp##.push-area-btn
 news.biglobe.ne.jp##.push-bx
+afkgaming.com##.zYmaG
+afkgaming.com#$#body.no-scroll { overflow: auto !important; }
```

Comment:
When you do CSS rules, they have to be CSS.
This `afkgaming.com#%#//scriptlet('set-local-storage-item', 'notifModal', 'true')` works fine.